---a/net/minecraft/entity/ai/EntityAICreeperSwell.java
+++b/net/minecraft/entity/ai/EntityAICreeperSwell.java
@@ -3,53 +3,44 @@
 import net.minecraft.entity.EntityLivingBase;
 import net.minecraft.entity.monster.EntityCreeper;
 
-public class EntityAICreeperSwell extends EntityAIBase
-{
+public class EntityAICreeperSwell extends EntityAIBase {
+
     EntityCreeper swellingCreeper;
-    EntityLivingBase creeperAttackTarget;
+    // EntityLivingBase creeperAttackTarget; // Spigot
     private static final String __OBFID = "CL_00001614";
 
-    public EntityAICreeperSwell(EntityCreeper p_i1655_1_)
-    {
+    public EntityAICreeperSwell(EntityCreeper p_i1655_1_) {
         this.swellingCreeper = p_i1655_1_;
         this.setMutexBits(1);
     }
 
-    public boolean shouldExecute()
-    {
+    public boolean shouldExecute() {
         EntityLivingBase entitylivingbase = this.swellingCreeper.getAttackTarget();
         return this.swellingCreeper.getCreeperState() > 0 || entitylivingbase != null && this.swellingCreeper.getDistanceSqToEntity(entitylivingbase) < 9.0D;
     }
 
-    public void startExecuting()
-    {
+    public void startExecuting() {
         this.swellingCreeper.getNavigator().clearPathEntity();
-        this.creeperAttackTarget = this.swellingCreeper.getAttackTarget();
+        // this.creeperAttackTarget = this.swellingCreeper.getAttackTarget(); // Spigot
     }
 
-    public void resetTask()
-    {
-        this.creeperAttackTarget = null;
+    public void resetTask() {
+        // this.creeperAttackTarget = null; // Spigot
     }
 
-    public void updateTask()
-    {
-        if (this.creeperAttackTarget == null)
-        {
+    public void updateTask() {
+        // Spigot start
+        EntityLivingBase b = this.swellingCreeper.getAttackTarget();
+        if (b == null) {
             this.swellingCreeper.setCreeperState(-1);
-        }
-        else if (this.swellingCreeper.getDistanceSqToEntity(this.creeperAttackTarget) > 49.0D)
-        {
+        } else if (this.swellingCreeper.getDistanceSqToEntity(b) > 49.0D) {
             this.swellingCreeper.setCreeperState(-1);
-        }
-        else if (!this.swellingCreeper.getEntitySenses().canSee(this.creeperAttackTarget))
-        {
+        } else if (!this.swellingCreeper.getEntitySenses().canSee(b)) {
             this.swellingCreeper.setCreeperState(-1);
-        }
-        else
-        {
+        } else {
             this.swellingCreeper.setCreeperState(1);
         }
+        // Spigot end
     }
-}
+}
